### PR TITLE
Review AI Test Toolkit Changes

### PR DESCRIPTION
**Pull request by StefanMaron**
In this PR I did address (hopfully) most of the comments I put in the pull request.
Its mostly guideline changes and readability improvements.

### CODE REVIEW

File: src/Tools/AI Test Toolkit/src/API/AITLogEntryAPI.Page.al [hunk 1]
```diff
@@ -109,6 +109,7 @@ page 149038 "AIT Log Entry API"
                 {
                     Caption = 'Output Data';
                 }
+                field(lastModifiedDateTime; Rec.SystemModifiedAt) { }
```
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/API/AITLogEntryAPI.Page.al]: Great addition. Could you add caption please to these fields?

File: src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al [hunk 2]
```diff
@@ -10,8 +10,10 @@ using System.TestTools.TestRunner;
 
 table 149034 "AIT Log Entry"
 {
+    Caption = 'AIT Log Entry';
```
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al]: Lets not put AIT to the Caption. Recommendation - AI Test Log entry.

File: src/Tools/AI Test Toolkit/src/Logs/AITTestMethodLinesCompare.Page.al [hunk 3]
```diff
@@ -7,6 +7,7 @@ namespace System.TestTools.AITestToolkit;
 
 page 149035 "AIT Test Method Lines Compare"
 {
+    Caption = 'AIT Test Method Lines Compare';
```
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/Logs/AITTestMethodLinesCompare.Page.al]: Same comment here 
AI Test Method...

File: src/Tools/AI Test Toolkit/src/Logs/AITTestSuiteCompare.Page.al [hunk 4]
```diff
@@ -7,6 +7,7 @@ namespace System.TestTools.AITestToolkit;
 
 page 149036 "AIT Test Suite Compare"
 {
+    Caption = 'AIT Test Suite Compare';
```
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/Logs/AITTestSuiteCompare.Page.al]: AI Test Suite

File: src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLine.Table.al [hunk 5]
```diff
@@ -10,6 +10,7 @@ using System.TestTools.TestRunner;
 
 table 149032 "AIT Test Method Line"
 {
+    Caption = 'AIT Test Method Line';
```
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLine.Table.al]: AI Test Method Line

File: src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al [hunk 6]
```diff
@@ -9,6 +9,7 @@ using System.TestTools.TestRunner;
 
 table 149030 "AIT Test Suite"
 {
+    Caption = 'AIT Test Suite';
```
Comment (nikolakukrika) [inline src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al]: AI Test Suite

File: src/Tools/Test Framework/Test Runner/src/TestMethodLine.Table.al [hunk 7]
```diff
@@ -127,6 +127,13 @@ table 130450 "Test Method Line"
         {
             DataClassification = CustomerContent;
         }
+        field(56; "No. of Functions"; Integer)
+        {
+            FieldClass = FlowField;
+            Caption = 'No. of Functions';
+            CalcFormula = count("Test Method Line" where("Test Suite" = field("Test Suite"), "Test Codeunit" = field("Test Codeunit"), "Line Type" = const(Function)));
+            Editable = false;
```
Comment (nikolakukrika) [inline src/Tools/Test Framework/Test Runner/src/TestMethodLine.Table.al:135]: I think these fields are not editable when surfaced in client, but no harm to be explicit.

**Review by nikolakukrika — APPROVED** [review]
Thanks a lot for doing this change. Could you update the minor comments?
If not we will fix it after it is merged.  Feel free to merge it when you can.

**Comment by StefanMaron** [comment]
@nikolakukrika I can not merge it myself, but I did the changes

**Review by nikolakukrika — APPROVED** [review]

**Review by Aleyenda — APPROVED** [review]

**Comment by nikolakukrika** [comment]
We had conflicts, I have resolved them in a different branch and merged.
Thanks a lot for the review and the fixes!

**Pull request closed without merging**